tests/playwright/Report.md

```diff
@@ -0,0 +1,176 @@
+        await jestPlaywright.resetBrowser()
+    })
+```
+
+Debug helper functions. This help in debugging test incase there are test failure. It uses the `jestPlaywrightDebug()`
+
+Example usage of Debug
+
+```
+    test.jestPlaywrightDebug('failed', async ({ page }) => {
+        await page.goto('https://www.wikipedia.org/')
+        const title = await page.title()
+        await expect(title).toBe('Nothing')
+    })
+```
+
+For more of these gems that come with jest-playwright, please visit the [Github Repo](https://github.com/playwright-community/jest-playwright)
+
+#### Playwright and WebdriverIO metrics
+
+**Statics on  WebdriverIO and Playwright Popularity**
+
+![Screenshot](statistics.png)
+
+The above data was adopted from [npm trends](https://www.npmtrends.com/cypress-vs-playwright-vs-puppeteer-vs-selenium-vs-testcafe) showing some of the most popular automation test tools in terms of popularity. From the above charts, we see that WebdriverIO still trumps Playwright when it comes to popularity among the testing community.
+
+**WebdriverIO and Playwright Mediawiki-core Tests Analysis**
+
+![Screenshot](evaluation-chart.png)
+
+The above visaulised chart represents data that was collected by running mediawiki/core tests in WebdriverIO and Playwright. A total of 14 tests were run using both frameworks on [Github Actions](https://docs.github.com/en/actions) platform.
+
+The tests were run 40 times in both framework to ascertain stability and reliability. No flakiness and failures were encountered during the tests run. As shown above, there is consitency in time when the tests were run in WebdriverIO and Playwright.
+
+The above chart further shows that Playwright is much faster than WebdriverIO in terms of speed.
+ 
+### Why adapt Playwright
+Playwright offers some great reasons such as;
+- It's simple to set up
+- Stable features. 
+- Ability to install Chrome, Firefox or WebKit (Safari) automatically
+- Bidirectional (events) – automating things like console logs is easy
+- Maintained by Microsoft people with experience maintaining Puppeteer
+
+However, it also presents some cons such as; 
+- It is very new so the APIs are evolving
+- Has no support for IE11 or non-browser platforms
+- Documentations and community are not as good as the other framework yet.
+### Advantages and disadvantages of Migrating from WebdriverIO
+From the various evaluations done against WebdriverIO, there seems to be great potential replacement and, Puppeteer and Microsoft Playwright seem to stand out in terms of both speed, Stability. However, the question being asked is, is it worth migrating from WebdriverIO to another test automation framework? Let’s look at the Advantages and disadvantages of a potential migration;
+
+*Advantages*
+- WebdriverIO is Javascript based and is  built over nodeJs just like Puppeteer and Microsoft Playwright thus it will be easier for developers since there would be no differences in programming language.
```

Comment:
```suggestion
- WebdriverIO is Javascript based and is  built over Node.js just like Puppeteer and Microsoft Playwright thus it will be easier for developers since there would be no differences in programming language.
```

Node.js , according to https://en.wikipedia.org/wiki/Node.js.